Poker/Deck.cs

```diff
@@ -24,25 +35,25 @@ private void CreateDeck()
                 foreach (CardRank rank in Enum.GetValues(typeof(CardRank)))
                 {
                     Card card = new Card(suit, rank);
-                    listOfCards.Add(card);
+                    ListOfCards.Add(card);
                 }
             }
         }
         
 
-        private void Shuffle(List<Card> listOfCards)
+        private void Shuffle()
```

Comment:
Removed the parameter which was not needed since the property/field belongs to the class